index.html

```diff
@@ -9730,9 +9730,8 @@ <h2>User prompts</h2>
 
 <p><a>User prompts</a> that are spawned
  from <a><code>beforeunload</code></a> event handlers,
- are <a>dismissed</a> implicitly upon <a>navigation</a>
- or <a>close window</a>,
- regardless of the defined <a>user prompt handler</a>.
+ are <a>dismissed</a> implicitly upon a <a data-lt="Navigate To">navigation</a>
```

Comment:
#1040 changed `go` to navigate to and broke the link.
#1518 then tried to fix warnings in invalid links but linked to the wrong thing.